frontend/apps/reader/modules/readersearch.lua

```diff
@@ -39,6 +39,9 @@ function ReaderSearch:onShowFulltextSearchInput()
 
     self.input_dialog = InputDialog:new{
         title = _("Enter text to search for"),
+        input = self.last_fulltext,
```

Comment:
`self.last_search_input` or `self.last_search_text` (as "fulltext" is only used for the menu item id and text).